rpm/python3.spec

```diff
@@ -99,6 +100,7 @@ Authors:
 
 %package -n python3-devel
 Requires:       %{name} = %{version}
+Requires:       pkgconfig(libcrypt)
```

Comment:
Shouldn't python's .pc file automatically depend on this if it really needs this?